fs/fs.go

```diff
@@ -113,7 +115,12 @@ func NewFsInfo(context Context) (FsInfo, error) {
 		}
 	}
 
+	// need to call this before the log line below printing out the partitions, as this function may
+	// add a "partition" for devicemapper to fsInfo.partitions
+	fsInfo.addDockerImagesLabel(context, mounts)
```

Comment:
Why are we running this method twice. Instead why not move all the label additions to this line?